source/index.rst

```diff
@@ -20,8 +20,12 @@ are roughly designed as single, guided courses.  Hands-on SciComp
 links to all of these materials and organizes them for self-study and
 determining where you need to start and where you need to end up.
 
+This course is coordinated by `Aalto University Science-IT`_  (See `About`_ for contact info)
```

Comment:
```suggestion
This course is coordinated by `Aalto University Science-IT`_  (See :doc:`about` for contact info)
```

Reply:
RST can directly refer to other documents like this (link name becomes the title)